enterprise/cli/licenses.go

```diff
@@ -187,3 +192,20 @@ func (r *RootCmd) licenseDelete() *clibase.Cmd {
 	}
 	return cmd
 }
+
+func convertLicenseExpireTime(license codersdk.License) codersdk.License {
+	if license.Claims["license_expires"] != nil {
+		value, ok := license.Claims["license_expires"].(json.Number)
```

Comment:
Curious: did you try to go directly with `license.Claims["license_expires"].(float64)`?